python/resdata/util/util/version.py

```diff
@@ -91,11 +91,10 @@ def getBuildTime(self):
     def getGitCommit(self, short=False):
         if self.git_commit is None:
             return "???????"
+        elif short:
```

Comment:
This function is never called with short=False in the tests.

Reply:
Yes, but maybe we can just leave this file as is for now and make an issue to redo the `util/util/version.py` debacle.